reference/configuration/framework.rst

```diff
@@ -389,12 +389,15 @@ form
 enabled
 .......
 
-**type**: ``boolean`` **default**: ``false``
+**type**: ``boolean`` **default**: (see explanation below)
 
 Whether to enable the form services or not in the service container. If
 you don't use forms, setting this to ``false`` may increase your application's
 performance because less services will be loaded into the container.
 
+The default value is ``true`` when the project uses ``symfony/symfony`` as a
+dependency. Otherwise, it defaults to ``false``.
```

Comment:
This is incorrect. The default is still disabled. The only case where it is automatically enabled is when you're using `symfony/flex` *and* `symfony/form` is required as well.

This also applies to assets, validation, translation, serialization and csrf. Can we add the note there as well?

Reply:
Indeed, issue updated accordingly

Reply:
Hmm. So I think explaining the logic behind the default value is too complex and doesn't add much value. A more useful thing is to run `debug:config framework`. I think we should mention this command at the top of each reference, and remove the `default` value in cases where there is not a simple answer.